frontend/src/slices/auth/actions.ts

```diff
@@ -55,11 +55,17 @@ const getCurrentUser = createAsyncThunk<
   const token = await storage.get(StorageKey.TOKEN);
   const hasToken = Boolean(token);
 
-  if (hasToken) {
-    return await authApi.getCurrent();
+  if (!hasToken) {
+    return null;
   }
 
-  return null;
+  try {
+    return await authApi.getCurrent();
```

Comment:
On `getCurrent` request from the backend side it throws an error from query builder with incorrect `where` param of `undefined` (because there is no proper `userId` with incorrect token) - you should check on BE get current user method if `userId` exists and in case if it's not - throw an error before it makes a query to the db

Reply:
![image](https://user-images.githubusercontent.com/87016333/227949883-087949eb-ae57-4b73-b932-3dc5cf225297.png)
We also have calling user service method "find" in our autorization plugin, so that this error occurs in the plugin and after objection throws an error it doesn't reach controller merthod getCurrent

Reply:
Yes, I am about that error,
And yep, let's check if decoded `userId` actually valid for fetching user